spec/factories/user_factory.rb

```diff
@@ -0,0 +1,15 @@
+FactoryGirl.define do
+
+  factory :user do
+    name                  'Person'
+    username              'pers0n'
+    email                 'person@mail.com'
+    password              'bananas!'
+    password_confirmation 'bananas!'
+  end
+
+  factory :confirmed_user, :parent => :user do
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)